cardano-crypto-tests/src/Test/Crypto/VRF.hs

```diff
@@ -26,10 +26,15 @@ import Test.QuickCheck
          ((==>), (===), Arbitrary(..), Gen, Property,  NonNegative(..),
           counterexample)
 import Test.Tasty (TestTree, testGroup)
-import Test.Tasty.QuickCheck (testProperty)
+import Test.Tasty.QuickCheck (testProperty, vectorOf)
 
 {- HLINT IGNORE "Use <$>" -}
 
+--
+-- New type to generate random ByteStrings of size 32
```

Comment:
I'd spell it as "newtype" rather than "new type" (we specifically need this to be a Haskell `newtype` construct, not just a new type in the more general sense).